### ISSUE TITLE
[BUG] Unable to retrieve reflected ReturnType name from method

### ISSUE THREAD

**Issue by M0n7y5** [beefytech/Beef#1121]
I am unable to get name of a return type from reflected method info. Return type is enum. 
I was unable to reproduce it in different project, but the code is public on github. 
You can get it here: (branch ReflectionBug)
https://github.com/M0n7y5/SourceBeef/tree/ReflectionBug

Simply download workspace and launch InterfaceGenTest project.

**Comment by bfiete** [comment]
What's the failure I'm supposed to see here? I see the following and then I hit the `Console.Read()`
```
Generating VTable from IServerPluginCallbacks

INDEX: 0        Returns bool    Load
INDEX: 1        Returns void    Unload
INDEX: 2        Returns void    Pause
INDEX: 3        Returns void    UnPause
INDEX: 4        Returns char8*  GetPluginDescription
INDEX: 5        Returns void    LevelInit
INDEX: 6        Returns void    ServerActivate
INDEX: 7        Returns void    GameFrame
INDEX: 8        Returns void    LevelShutdown
INDEX: 9        Returns void    ClientActive
INDEX: 10       Returns void    ClientFullyConnect
INDEX: 11       Returns void    ClientDisconnect
INDEX: 12       Returns void    ClientPutInServer
INDEX: 13       Returns void    SetClientCommand
INDEX: 14       Returns void    ClientSettingsChanged
INDEX: 15       Returns null    ClientConnect
INDEX: 16       Returns null    ClientCommand
INDEX: 17       Returns null    NetworkIDValidated
INDEX: 18       Returns void    OnQueryCvarValueFinished
INDEX: 19       Returns void    OnEdictAllocated
INDEX: 20       Returns void    OnEdictFreed
INDEX: 21       Returns bool    BNetworkCryptKeyCheckRequired
INDEX: 22       Returns bool    BNetworkCryptKeyValidate
```

**Comment by bfiete** [comment]
Ah, I guess the `Returns null` is the error.

**Comment by bfiete** [comment]
Fixed at f0d99a2bcc9feb39c377fdb64107703bacc90e03
